### PR TITLE
Update Changelog for 1.11.5

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

**Review by guillep2k — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Comment by 6543** [comment]
I'll leave the decision to you - if #11349 should go into 1.11.5 or next one

**Comment by jolheiser** [comment]
Can you also update this version?
 https://github.com/go-gitea/gitea/blob/b09ec508c3f33e05d3784f96093451c463600d7b/docs/config.yaml#L21

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,9 +4,12 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
-## [1.11.5](https://github.com/go-gitea/gitea/releases/tag/v1.11.5) - 2020-05-08
+## [1.11.5](https://github.com/go-gitea/gitea/releases/tag/v1.11.5) - 2020-05-09
 
 * BUGFIXES
+  * Prevent timer leaks in Workerpool and others (#11333) (#11340)
```
Comment (zeripath) [inline CHANGELOG.md:10]:
```suggestion
  * Prevent timer leaks in Workerpool and others (#11333) (#11340)
  * Fix tracked time issues (#11349) (#11354)
```

**Review by zeripath — APPROVED** [review]

**Comment by zeripath** [comment]
make lg-tm work

**Comment by 6543** [comment]
🚀

**Review by 6543 — COMMENTED** [review]
lgtm

**Pull request merged**